src/scrape_up/MediEncyclopedia/mediencyclopedia.py

```diff
@@ -0,0 +1,96 @@
+from html.parser import HTMLParser
+import bs4
+from bs4 import BeautifulSoup
+import requests
+
+"""
+        Create an instance of `MediEncyclopedia` class.
+
+    ```python
+    ency = MediEncyclopedia()
+    ```
+
+    | Methods               | Details                                                                                      |
+    | --------------------- | -------------------------------------------------------------------------------------------- |
+    | `.scrapebyurl()`      | Returns the medical dictation of associated topic url                                        |
+    | `.query()`            | It takes a user query parameter as an argument and returns all relevant terms related to it. |
+    | `.byletter()`         | Returns the list of medical relics starting with a particular letter                         |
+
+
+    """
+
+
+class MediEncyclopedia:
+
+  def __init__(self):
+    print(self)
+
+  def scrapebyurl(self,url):
+    """Returns the medical data including references, review dates, content, topic-head and so on.\n
+    Parameters:\n
+      - url: It is the url of the Mediline's encyclopedia page the information of whom is to be fetched."""
+
+    try:
+      content = requests.get(url)
+      soup = BeautifulSoup(content.content, 'html.parser')
+      headline = soup.find('h1',attrs={'class':'with-also'}).text
+      article = soup.find('article')
+      for script in article(["script", "style"]):
+        script.extract()
+      text = article.get_text().replace('Browse the Encyclopedia','').replace('Clinical Trials','')
+      text = text.replace('To use the sharing features on this page, please enable JavaScript.','')
+
+      return [headline,text]
+    except:
+      return None
+
+  def query(self, userquery):
+    """This function takes a query from the users and returns all the matching terms/responses from the encyclopedia matching the user's query. If there is no matching response, than it returns None\n
```

Comment:
Again you have added statements to docstring, this is not the way to write a docstring. 
 
The docstring required for the package is as shown in the image below. Make changes similar to this.
![image](https://github.com/Clueless-Community/scrape-up/assets/93156825/125cac69-2a40-455a-9ab6-9f35e6ba74a4)